chapters/01-valores-tipos-operadores.md

```diff
@@ -87,26 +87,24 @@ Existe mais um operador aritmético que você talvez não reconheça imediatamen
 
 ## Números Especiais
 
-Existem 3 valores especiais no JavaScript que são considerados números, mas não comportam-se como números normais.
+Existem três valores especiais no JavaScript que são considerados números, mas não se comportam como números normais.
 
-Os dois primeiros são `Infinity` e `-Infinity`, que são usados para representar os infinitos positivo e negativo. `Infinity - 1` continua sendo `Infinity`, e assim por diante. Mas não ponha muita confiança neste tipo de computação *baseada em infinito*, pois é uma matemática pesada, e vai rapidamente levar para nosso próximo número especial: `NaN`.
+Os dois primeiros são `Infinity` e `-Infinity`, que são usados para representar os infinitos positivo e negativo. O cálculo `Infinity - 1` continua sendo `Infinity`, assim como qualquer outra variação dessa conta. Entretanto, não confie muito em cálculos baseados no valor infinito, pois esse valor não é matematicamente sólido, e rapidamente nos levará ao próximo número especial: `NaN`.
 
-`NaN` significa "not a number" (não é um número). Você obtém isso quando declara `0 / 0` (zero dividido por zero), `Infinity - Infinity`, ou qualquer número de outra operação numérica que não produz um preciso e significante valor.
+`NaN` é a abreviação de “_not a number_” (não é um número), mesmo sabendo que ele é um valor do tipo número. Você receberá esse valor como resultado quando, por exemplo, tentar calcular `0 / 0` (zero dividido por zero), `Infinity - Infinity` ou, então, realizar quaisquer outras operações numéricas que não resultem em um número preciso e significativo.
 
 ## Strings
 
-O próximo tipo básico de dado é a *string*. Strings são usadas para representar texto. Elas são escritas delimitando seu conteúdo entre aspas:
-
-```javascript
+O próximo tipo básico de dado é a _string_. _Strings_ são usadas para representar texto, e são escritas delimitando o seu conteúdo entre aspas.
 
+```js
 "Patch my boat with chewing gum"
 'Monkeys wave goodbye'
-
 ```
 
-Ambas as aspas simples e duplas podem ser usadas para marcar strings - contanto que as aspas no início e no fim da string combinem.
+Ambas as aspas simples e duplas podem ser usadas para representar _strings_, contanto que as aspas abertas no início e fechadas fim sejam iguais.
```

Comment:
Bom, se não for ter revisão, então ao mesmo tempo que a pessoa fizer o PR ela já pode dar o merge.

Como você não fez o merge na sequência, tomei a liberdade de fazer o review.

Minha opinião é que haja sempre um review **antes** do merge, pois outras pessoas podem ter opiniões diferentes se o que foi alterado é realmente melhor do que o novo texto.

E se nesse review forem detectados erros, eles devem ser corrigidos antes do merge.

Sobre a contribuição de outras pessoas, alguém que estiver lendo o texto pode ver um erro e apenas abrir uma issue mostrando o bug, para outra pessoa consertar, isso já seria de grande ajuda!